snaplet-sqlite-simple.cabal

```diff
@@ -1,5 +1,5 @@
 name:           snaplet-sqlite-simple
-version:        0.4.6
+version:        0.6.0.1
```

Comment:
No, it's not on hackage, I want to have working version in master and them upload to hackage.